pytrickle/stream_processor.py

```diff
@@ -105,7 +128,7 @@ def load_model(self, **kwargs):
         """Load model using provided function."""
         if self.model_loader:
             try:
-                self.model_loader(**kwargs)
+                self.model_loader(self, **kwargs)
```

Comment:
why do we need a reference to the processor? If its None won't this function be gone?